### ISSUE TITLE
CWCryptoWatch class: figure out better way of handling sql queries in python, either the psycops2.sql.SQL function, or maybe moving queries into postgres functions

### ISSUE THREAD

**Issue by psykoholik** [psykoholik/cryptowatch#2]
some of the methods for formatting queries didn't work as read, examples like 

query = "SELECT {} FROM {} WHERE"
query.format(columns, tablename) <-- didn't insert information correctly

also the %s syntax for psycopg2 really only works for strings like

query = "INSERT INTO TABLENAME VALUES ( %s, %s )"
cursor.execute(query, (string1, string2)) <-- works fine for these cases, but not the above cases

basically need to investigate best way to really manage sql queries in python
